cmd/rest-server/main.go

```diff
@@ -40,7 +40,7 @@ func init() {
 	flags.StringVar(&cpuProfile, "cpu-profile", cpuProfile, "write CPU profile to file")
 	flags.BoolVar(&server.Debug, "debug", server.Debug, "output debug messages")
 	flags.StringVar(&server.Listen, "listen", server.Listen, "listen address")
-	flags.StringVar(&server.Log, "log", server.Log, "log HTTP requests in the combined log format")
+	flags.StringVar(&server.Log, "log", server.Log, "write HTTP requests in the combined log format to the specified filename")
```

Comment:
Please put `filename` in "`" backticks. Then the command line will show as `--log filename`.

Reply:
Done!